requirements.txt

```diff
@@ -11,3 +11,4 @@ docker>=3.4.0,<4.0
 pandas>=0.22,<1.0
 scikit-learn>=0.19,<1.0
 tqdm>=4.20,<5.0
+tensorflow>=1.0,<2.0
```

Comment:
So we do not require Tensorflow because there are two versions: gpu and non-gpu and they are in conflict.

Reply:
CI installs it separately.